### PR TITLE
12.0 mig sales team security

### PR DESCRIPTION
**Pull request by Montxu**
cc @guadaltech

### CODE REVIEW

**Review by anajuaristi — APPROVED** [review]
LGTM :+1:

**Review by oihane — APPROVED** [review]
LGTM

File: sales_team_security/models/res_partner.py [hunk 1]
```diff
@@ -11,7 +11,8 @@ class ResPartner(models.Model):
     @api.model
     def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
                         submenu=False):
-        """Patch view to inject the default value for the team_id and user_id.
+        """
+        Patch view to inject the default value for the team_id and user_id.
```
Comment (pedrobaeza) [inline sales_team_security/models/res_partner.py:15]: Why changing this? docstring can start with the triple quote and the first doc line in the same line

File: sales_team_security/models/res_partner.py [hunk 2]
```diff
@@ -11,7 +11,8 @@ class ResPartner(models.Model):
...
         """
         # FIXME: Use base_view_inheritance_extension when available
```
Comment (pedrobaeza) [inline sales_team_security/models/res_partner.py:17]: Please consider this for simplifying a lot the code.

**Review by pedrobaeza — DISMISSED** [review]
For avoiding to be merged until questions are answered.

File: sales_team_security/models/res_partner.py (hunk 2 above)
Comment (Montxu) [reply sales_team_security/models/res_partner.py:17]: hi, this module (base_view_inheritance_extension) not in version 12.0, as soon this module is migrated i fix this.

Comment (pedrobaeza) [reply sales_team_security/models/res_partner.py:17]: We should push for having it migrated as if we merge this one without the dependency, then we won't be able to change the dependency in this branch due to stability policy.

Comment (pedrobaeza) [reply sales_team_security/models/res_partner.py:17]: As the other module is not merged and this solution only implies a bit more of code, I'm dismissing it and we go on.

**Pull request merged**